### DISCUSSION TITLE
分享,入手了一个HiFi入门耳机静海

### DISCUSSION THREAD

**Discussion by YMZGymzg** [share121/inter-knot#5205] [其他]
初步体验下来感觉很好，高中低频比较均衡，对于我这个新手来说合适

**Comment by March-777th** [comment]
同款耳机<img src="https://img-static.mihoyo.com/communityweb/upload/d41b52daa44ef813710cf4b482a256b2.png" alt="三月七_点赞" width="75" >

**Comment by Fluorine-z** [comment]
耳机没买，但是买了手柄，八爪鱼4，玩了几天爽炸了
